pkg-plist

```diff
@@ -1,4 +1,5 @@
 bin/photoprism
+etc/rc.d/photoprism
```

Comment:
This is unnecessary, this line in the Makefile already does the job:
https://github.com/huo-ju/photoprism-freebsd-port/pull/73/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R36

Reply:
yeah, no need to copy etc/rc.d/*

Reply:
this is generated by `make pkglist`, should i remove it maunally?

Reply:
Yes, it does generate there because the port has the file, but in the port's Makefile that file is defined as an RC service, so there's no need to list it here.

Reply:
@Gaojianli  I have removed it on the main branch. No worries, and thank you!